src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -69,7 +70,7 @@ private static CommandSpec ResolveFromRootedCommand(string commandName, IEnumera
 
             if (commandPackage == null) return null;
 
-            var depsPath = projectContext.GetOutputPathCalculator().GetDepsPath(Constants.DefaultConfiguration);
+            var depsPath = projectContext.GetOutputPathCalculator(Constants.DefaultConfiguration).RuntimeFiles.Deps;
```

Comment:
Canged CompilationFiles  and RuntimeFiles to methods GetCompilationFiles and GetRuntimeFiles